### PR TITLE
Fix/ubuntu log

### PR DESCRIPTION
**Pull request by Julien4218**

### CODE REVIEW

**Comment by Julien4218** [comment]
@jbeveland27 
I've tested the install of the ubuntu log recipe (not through guided install), here is the output:

ubuntu@ip-172-31-17-67:~$ curl -Ls https://raw.githubusercontent.com/newrelic/newrelic-cli/master/scripts/install.sh | bash && sudo NEW_RELIC_REGION=STAGING NEW_RELIC_API_KEY=NRAK-HBB0XXXXXXXXXXXXXXXX NEW_RELIC_ACCOUNT_ID=YYYYYYY /usr/local/bin/newrelic install -c https://raw.githubusercontent.com/newrelic/open-install-library/fix/ubuntu-log/recipes/newrelic/infrastructure/logs
/ubuntu-logs.yml
Starting installation.
Installing New Relic CLI v0.18.32
Installing to /usr/local/bin using sudo

   _   _                 ____      _ _
  | \ | | _____      __ |  _ \ ___| (_) ___
  |  \| |/ _ \ \ /\ / / | |_) / _ | | |/ __|
  | |\  |  __/\ V  V /  |  _ |  __| | | (__
  |_| \_|\___| \_/\_/   |_| \_\___|_|_|\___|

  Welcome to New Relic. Let's install some instrumentation.

  Questions? Read more about our installation process at
  https://docs.newrelic.com/


The following will be installed:
  Logs integration

==> Installing logs-integration...
Which log files would you like to tail? ¦var/log/alternatives.log,/var/log/cloud-init.log,/var/log/auth.log
    ? Checking for data in New Relic...success.
  ??  The Logs configuration file (base configuration) can be found in /etc/newrelic-infra/logging.d/logging.yml
  ??  The Logs configuration file for discovered processes can be found in /etc/newrelic-infra/logging.d/discovered.yml
  Edit these files to make changes or configure advanced features for the Logs integration. See the docs for options:
  https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent#parameters

==> Installing logs-integration...success.

  New Relic installation complete!
  Your data is available at https://one.newrelic.com/redirect/entity/MTA2MDYxODN8SU5GUkF8TkF8NzYwMzA2NDc3OTA2NjIzMjIxMQ
ubuntu@ip-172-31-17-67:~$

File: recipes/newrelic/infrastructure/logs/ubuntu-logs.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: logs-integration
+displayName: Logs integration
+description: New Relic install recipe for basic Logging via Infra-Agent
+repository: https://github.com/newrelic/infrastructure-agent/tree/master/assets/examples/logging
+
+installTargets:
+  - type: host
+    os: linux
```
Comment (jbeveland27) [inline recipes/newrelic/infrastructure/logs/ubuntu-logs.yml:11]: I think this should be more specific to `debian`/`ubuntu` platforms

**Review by jbeveland27 — CHANGES_REQUESTED** [review]

**Comment by jbeveland27** [comment]
This recipe looks good overall (once the `platform` is added to `installTargets`). However we need to review the CLI to know if it will pick up on this automatically for the correct distros with guided install.

File: recipes/newrelic/infrastructure/logs/ubuntu-logs.yml (hunk 1 above)
Comment (Julien4218) [reply recipes/newrelic/infrastructure/logs/ubuntu-logs.yml:11]: Ho this is a bug, thanks for catching it, I've just added the platform

**Comment by Julien4218** [comment]
> This recipe looks good overall (once the `platform` is added to `installTargets`). However we need to review the CLI to know if it will pick up on this automatically for the correct distros with guided install.

Correct, the CLI is passing the DiscoverManifest when fetching the logging recipe, so it should pickup the ubuntu log recipe https://github.com/newrelic/newrelic-cli/blob/master/internal/install/recipe_installer_guided.go#L31

**Pull request merged**